_posts/2022-04-28-cockpit-268.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cockpit 268
+author: garrett
+date: '2022-04-28'
+tags: cockpit
+slug: cockpit-268
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 268:
+
+
+## Software Updates: Only install kpatches
+
+Live patch a kernel without having to reboot or restart services with the new "Install kpatch updates" button. This allows patching of urgent kernel vulnerabilities without rebooting.
+
+![kpatch]({{ site.baseurl }}/images/268-software-updates-only-install-kpatches.png)
+
+
+## Try it out
+
+Cockpit 268 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/268)
```

Comment:
```suggestion
* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/268.1)
```

We have a hotfix.

Reply:
I think we're not going to mention the .1 anywhere else on the release notes though... right? Just have the updated 268.1 links, but still call it "268", since there are no functional changes (only the button order adjustment).